### PR TITLE
api/cli: update dependencies and Rust edition to 2021

### PR DESCRIPTION
**Pull request by mcobzarenco**

### CODE REVIEW

File: api/Cargo.toml [hunk 1]
```diff
@@ -1,28 +1,28 @@
 [package]
 name = "reinfer-client"
 version = "0.11.0"
-description = "API client for Re:infer"
+description = "API client the Re:infer conversational data intelligence platform"
```
Comment (jcalero) [inline api/Cargo.toml:4]: Not sure what you were going for here but this is an odd sentence.

"API client **for Re:infer** the conversational ..." maybe?

File: cli/Cargo.toml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 [package]
 name = "reinfer-cli"
 version = "0.11.0"
-description = "Command line interface for the reinfer platform."
+description = "Command line interface for the Re:infer conversational data intelligence platform"
```
Comment (jcalero) [inline cli/Cargo.toml:4]: Again, weird phrasing, what are you trying to say :sweat_smile:, what's "the Re:infer"?

Comment (jcalero) [inline cli/Cargo.toml:4]:
```suggestion
description = "Command line interface for Re:infer, the conversational data intelligence platform"
```

**Review by jcalero — APPROVED** [review]
All good, just find some of the phrasing with the package descriptions to be strange, so wondering if it's a mistake. Obviously minor nit.

File: api/Cargo.toml (hunk 1 above)
Comment (mcobzarenco) [reply api/Cargo.toml:4]: Ah yes, well spotted

File: cli/Cargo.toml (hunk 2 above)
Comment (mcobzarenco) [reply cli/Cargo.toml:4]: Hahaha, I just copy pasta'ed from somewhere else, apparently with no regard for grammer :sweat_smile:

Comment (mcobzarenco) [reply cli/Cargo.toml:4]: I think I was trying to say "for the Re:infer conversational data intelligence platform", not that we are THE one

**Comment by mcobzarenco** [comment]
Fixed the copy

**Comment by jcalero** [comment]
LGTM

**Pull request merged**